pvlib/singlediode_methods.py

```diff
@@ -0,0 +1,347 @@
+"""
+Calculate single diode model currents and voltages using methods from
+J.W. Bishop (Solar Cells, 1988).
+"""
+
+from functools import partial
+import numpy as np
+try:
+    from scipy.optimize import brentq
+except ImportError:
+    brentq = NotImplemented
+# FIXME: change this to newton when scipy-1.2 is released
+try:
+    from scipy.optimize import _array_newton
+except ImportError:
+    from pvlib import tools
```

Comment:
is ``tools`` used anywhere in this module? I don't see it... if not, I think the style guide says to remove the import.

Reply:
✔️